CMakeLists.txt

```diff
@@ -212,6 +204,19 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 source_group(TREE ${PROJECT_SOURCE_DIR}/include/mrdox PREFIX "include" FILES ${INCLUDES})
 source_group(TREE ${PROJECT_SOURCE_DIR}/source PREFIX "source" FILES ${SOURCES})
 
+#-------------------------------------------------
+#
+# Plugins
+#
+#-------------------------------------------------
+
+file(GLOB plugins LIST_DIRECTORIES 1 CONFIGURE_DEPENDS plugins/*)
```

Comment:
but it is called addons/ ?

Reply:
Wait no, that's correct - the source for the plugin (as of now) is in `/plugins/...` while it's build artifact (i.e. the dll) gets put into `addons/plugins`.